pages/privacy-policy.js

```diff
@@ -35,7 +35,7 @@ const privacyPolicy = () => {
                         </div>
                     </div>
                     <p id="1b" id="1" className={style.modal__button__para}>
-                        Vovoca is a project under team DevBucket. We have putted our heart and soul in it to bring it to you for free. We our free today because of several open-source technologies out there that we have used all togethor to bring you Vovoca. We therefore aim to bring you vovoca as a open-source program too after completion of all the basic structure and interface. Our backend APIs are currently hosted on heroku and frontend on vercel while in development mode.
+                    Vovoca is a project under team DevBucket. We have put our heart and soul into it to bring it to you for free. We are free today because of several open-source technologies out there that we have used all together to bring you Vovoca. We therefore aim to bring you Vovoca as a open-source program too after completion of all the basic structure and interface. Our backend APIs are currently hosted on Heroku and frontend on vercel while in development mode.
                     </p>
```

Comment:
Replace text with `Welcome to Vovoca, a remarkable endeavor by the passionate team at DevBucket. Our unwavering dedication has fueled the creation of this platform, offered to you completely free of charge. We owe our freedom today to the incredible open-source technologies that have seamlessly converged within Vovoca. With utmost commitment, our ultimate goal is to transform Vovoca into an open-source program, granting access to its inner workings once the foundation and interface are perfected. During our development journey, Vercel graciously hosts both our backend APIs and frontend, ensuring a seamless and efficient experience. Together, let's embrace the power of collaboration and embark on a harmonious musical voyage with Vovoca.`